apollo-runtime/src/main/java/com/apollographql/apollo/internal/response/RealResponseReader.java

```diff
@@ -199,7 +199,7 @@ public RealResponseReader(Operation.Variables operationVariables, R recordSet,
       resolveDelegate.didResolveNull();
       result = null;
     } else {
-      CustomTypeAdapter<T> typeAdapter = scalarTypeAdapters.adapterFor(field.scalarType());
+      CustomTypeAdapter<T> typeAdapter = scalarTypeAdapters.adapterFor(field.getScalarType());
```

Comment:
It would be good to hold off for the depreciation changes in bug fixes. They are because of Kotlin changes and will make it harder to cherry pick into 1.4.x branch.